ocs_ci/ocs/ui/deployment_ui.py

```diff
@@ -26,6 +27,11 @@ def __init__(self, driver):
         super().__init__(driver)
         self.ocp_version = get_ocp_version()
         self.dep_loc = locators[self.ocp_version]["deployment"]
+        self.operator = (
+            OCS_OPERATOR
+            if Version.coerce(self.ocp_version) < Version.coerce("4.9")
```

Comment:
ocs version should be used to select ODF/OCS operator.

Reply:
can you use approach introduced here:
https://github.com/red-hat-storage/ocs-ci/pull/4872/files

Also see info in this email:
https://mailman-int.corp.redhat.com/archives/ocs-qe/2021-September/msg00292.html

Reply:
done